android/src/main/java/com/instabug/reactlibrary/RNInstabugAPMModule.java

```diff
@@ -335,9 +335,30 @@ public void run() {
     *  Send Apm network log by Reflection
     */
     @ReactMethod
-    public void networkLog(String networkData) throws JSONException {
+    public void networkLog(String networkData, Boolean isW3cHeaderFound,Double partialId,Double networkStartTimeInSeconds,String W3CGeneratedHeader,String W3CCaughtHeader) throws JSONException {
         try{
             APMNetworkLogger apmNetworkLogger = new APMNetworkLogger();
+
+            Long NetworkStartTime = null;
+            Long PartialId=null;
+            String GeneratedHeader=null;
```

Comment:
Let's rename all the variables to use `camelCase` instead of `PascalCase`, for ones starting with `W3C` like `W3CGeneratedHeader` I think we can rename them to start with `w3c` like `w3cGeneratedHeader`.